lib/index.js

```diff
@@ -19,6 +19,7 @@ var defaults = {
   debug: identity,
   lint: true,
   minify: false,
+  encapsulateStyles: true,
```

Comment:
Okay, sounds good.

I think we'll need to add some documentation to the main SUIT docs to explain this as I expect this could be a bit puzzling at first and it's nice to explain the rationale.